src/javascript/app_2/Modules/Trading/Components/Form/Purchase/contract-info.jsx

```diff
@@ -58,12 +58,11 @@ const ContractInfo = ({
                 </div>
                 }
             </div>
-            <Tooltip
+            <Popover
                 alignment='left'
-                className={classNames('trade-container__price-tooltip', { 'trade-container__price-tooltip--disabled': has_error_or_not_loaded })}
-                classNameIcon='trade-container__price-tooltip-i'
                 icon='info'
-                message={has_error_or_not_loaded ? '' : proposal_info.message}
+                margin={210}
```

Comment:
this is margin from which side?, or its all side margins?

Reply:
margin of whatever side you set `left` will add n more px to the calculation